### ISSUE TITLE
Logging in issue

### ISSUE THREAD

**Issue by GPFusionFNYT** [ImpactDevelopment/ImpactIssues#3122]
Hi. I am a user who just downloaded impact. I am having trouble launching it. when I launch it, it signs me in as a default that only has demo access and cant play the actual game. can you please help. I did log in from an account that owns the game

**Comment by Callihan-ops** [comment]
I do not believe this is an issue directly related to impact but rather minecraft as a whole. There are some simple trouble shooting steps you could take, such as

1. Making sure you own minecraft
2. Restarting your computer
